tools/db_bench_tool.cc

```diff
@@ -4578,7 +4578,7 @@ void VerifyDBFromDB(std::string& truth_db_name) {
       if (FLAGS_max_scan_distance != 0) {
         if (FLAGS_reverse_iterator) {
           GenerateKeyFromInt(
-              (uint64_t)std::max(0l, seek_pos - FLAGS_max_scan_distance),
+              (uint64_t)std::max(0ll, seek_pos - FLAGS_max_scan_distance),
```

Comment:
in travis this change is causing:

```
tools/db_bench_tool.cc:4581:73: error: no matching function for call to ‘max(long long int, int64_t)’
               (uint64_t)std::max(0ll, seek_pos - FLAGS_max_scan_distance),
```

It's very confusing to me. Maybe just use `static_cast<int64_t>(0)`

Reply:
sure